check-vulnerabilities/action.yml

```diff
@@ -453,7 +462,11 @@ runs:
     - name: "Run safety advisory checks"
       shell: bash
       run: |
-        python dependency-check.py
+        if [[ ${{ inputs.hide-log }} == 'true' ]]; then
+          python dependency-check.py > /dev/null 2>&1

```

Comment:
@RobPasMue @jorgepiloto Maybe we should just redirect `stdout` and not `stderr`for any potential debugging.
But I don't have any certainty of futur content of the `stderr`.

Reply:
I'd redirect everything honestly. Don't worry

Reply:
Since this PR is about hidding output logs, which make total sense, I think we should redirect to `stdout` for the moment. Like you, I'm not sure about the content of `stderr` either. Moreover, the "core" of this action is related to a python script written on the fly. I have the feeling that we could debug manually if necessary.